<?xml version="1.0"?>
<launch>
    <!--2. 各执行器驱动-->

    <!--机械臂末端电爪驱动-->
    <node name="gripper1_driver" pkg="gripper_controllers" type="gripper1_driver"/>
    <node name="gripper2_driver" pkg="gripper_controllers" type="gripper2_driver" output="screen"/>

    <!--操纵杆检测设备驱动-->
    <node name="airhandle_server" pkg="air_handle" type="airhandle_server" output="screen"/>

</launch>
